python3/raygun4py/raygunprovider.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, api_key, config={}):
             )
 
         try:
-            import ssl
+            import ssl  # noqa: F401
```

Comment:
F401 = unused import

Reply:
Add as a comment to cover for documentation ^